flink-end-to-end-tests/flink-stream-state-ttl-test/src/main/java/org/apache/flink/streaming/tests/MonotonicTTLTimeProvider.java

```diff
@@ -54,14 +57,24 @@
 	private static final Object lock = new Object();
 
 	@GuardedBy("lock")
-	static long freeze() {
+	static <T, E extends Throwable> T doWithFrozenTime(FunctionWithException<Long, T, E> action) throws E {
```

Comment:
I moved this method up because in our code style we try to adhere to the following method order in the class:
```

static fields

non-static fields

constructor1() { constructor2() }

constructor2() { }

method1() { method2() }

method2() { method3() }

method3() { }

staticMethod1() { staticMethod2(); staticMethod3(); }

staticMethod2() { }

staticMethod3() { }

inner classes

embedded static classes

...

```

Reply:
Thanks for the explanation! I'll follow this style guide from now on.